### ISSUE TITLE
defusedxml.ElementTree breaks the xml.etree.ElementTree package

### ISSUE THREAD

**Issue by Gittenburg** [tiran/defusedxml#54]
This has already been reported as #43 and #48. I am opening yet another issue for this because the previous issues describe symptoms instead of the underlying problem and I want to spare others from debugging for hours, like I just did.

@marienz has opened the PR #53 to fix this, and provided the following minimal breaking example:

```py
from xml.etree import ElementTree as FirstElementTree
from defusedxml import ElementTree as DefusedElementTree
from xml.etree import ElementTree as SecondElementTree
assert FirstElementTree is SecondElementTree
```

Attempting to append an element from the `FirstElementTree` to the `SecondElementTree` yields the error described in #43:

```
TypeError: append() argument must be xml.etree.ElementTree.Element, not Element
```

This bug is bound to affect any larger Python project that uses both `defusedxml` and [python-markdown](https://github.com/Python-Markdown/markdown/) (see python-markdown/markdown/issues/950). Examples of projects affected by this are Jupyter (jupyter/nbviewer/issues/909, danielfrg/pelican-jupyter/issues/124) and [Zulip](https://github.com/zulip/zulip/).

@tiran would you mind merging #53 and releasing a new version?  Considering that [CPython's etree is still vulnerable](https://github.com/python/cpython/blob/master/Doc/library/xml.rst#xml-vulnerabilities) to billion laughs and quadratic blowup.

**Comment by johnmathews** [comment]
@Gittenburg did you find a fix for this? I ran into this issue today and haven't been able to fix it.

**Comment by marienz** [comment]
@johnmathews if applying the fix from #53 to your copy of defusedxml doesn't work for you, and you can provide instructions that allow me to reproduce the problem, I can have a look. Note that in addition to failing code we may need to know which versions of ElementTree-related libraries you're using and where they're installed, as it is possible (although tricky) to import multiple ElementTree libraries (or the same library under different import paths) and trigger a very similar error that way.
